addons/website_sale/views/templates.xml

```diff
@@ -403,23 +404,39 @@
         </li>
     </template>
 
+    <template id="categorie_link" name="Category Link">
+        <div t-attf-onclick="location.href='#{keep('/shop/category/' + slug(c), category=0)}';" class="custom-control custom-radio mb-1 d-inline-block">
+            <input type="radio" style="pointer-events:none;" class="custom-control-input" t-att-id="c.id" t-att-value="c.id" t-att-checked="'true' if c.id == category.id else None"/>
+            <label class="custom-control-label font-weight-normal" t-att-for="c.id" t-field="c.name"/>
+        </div>
+    </template>
```

Comment:
this is the only solution I found to mix input radio and link.. I hope that it's right for you